src/app/modules/trade-companion/component/trade-notifications-panel/trade-notifications-panel.component.ts

```diff
@@ -222,6 +222,11 @@ export class TradeNotificationPanelComponent implements OnInit, AfterViewInit, O
     this.closeClick$.next()
   }
 
+  public deleteAll(): void {
+    this.notifications = []
```

Comment:
Looking at the `onDismissNotification` method again, it seems like it does some additional steps which are probably also required here:
```
this.tradeNotificationsService.dismissNotification(notification)
this.updateActiveNotificationIndex()
this.ref.detectChanges()
```
Probably you'd have to loop over each notification and let the service know it has been dismissed.

Reply:
Sounds good. I'll take a look at cleaning this up once I have time after the league winds down.